examples/frustum-cull/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-basic",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-dev-server --progress --hot --open",
+    "start-local": "webpack-dev-server --env.local --progress --hot --open",
+    "build": "webpack -p"
+  },
+  "dependencies": {
+    "deck.gl": "^7.0.0-beta",
+    "luma.gl": "^7.0.0-beta",
```

Comment:
This is unnecessary. You should be able to import the geometry from `@luma.gl/core` which is already a deck dependency.